src/app/staking/components/PanelNewDeposit.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import React from "react";
+import Image from "next/image";
+import Tenant from "@/lib/tenant/tenant";
+import { useRouter } from "next/navigation";
+import { useAgoraContext } from "@/contexts/AgoraContext";
+
+export const PanelNewDeposit = () => {
+  const { isConnected } = useAgoraContext();
```

Comment:
is this for wallet connection? Is there any reason for using this vs wagmi `useAccount`? I prefer using the wagmi hooks unless theres a good reason for it, otherwise it just feels like an additional layer of abstraction